components/brave_wallet/browser/zcash/zcash_tx_meta.cc

```diff
@@ -35,11 +35,13 @@ mojom::ZecTxDataPtr ToZecTxData(const std::string& chain_id,
           mojom::ZecTxOutput::New(*orchard_unified_addr, output.value));
     }
   }
+
   // TODO(cypt4): Add proper flag here
   // https://github.com/brave/brave-browser/issues/39314
```

Comment:
Do we still need a flag here? I see the issue linked here is closed now

Reply:
Removed